### DISCUSSION TITLE
QR Command to set GoPro Hero12 video profile (Standard, HDR, LOG)

### DISCUSSION THREAD

**Discussion by aroooy** [gopro/labs#565] [Q&A] [answered]
Could you please tell me the QR Command to set the video profile (Standard, HDR, LOG) of GoPro Hero12?  

I tried looking for documentation, QR generation, etc., but I couldn't find anything.

**Comment by dnewman-gpsw** [comment, marked as answer]
It is in the release notes.

mS - standard 
mL - Log
mH - HDR

**Reply by aroooy** [reply]
Thank you very much. 
 I'll check the release notes.
